sky/cli.py

```diff
@@ -3525,11 +3525,10 @@ def storage_delete(names: List[str], all: bool, yes: bool):  # pylint: disable=r
     if sum([len(names) > 0, all]) != 1:
         raise click.UsageError('Either --all or a name must be specified.')
     if all:
-        storages = sky.storage_ls()
-        if not storages:
+        names = global_user_state.get_glob_storage_name('*')
```

Comment:
```suggestion
        # Use '*' to get all storages.
        names = global_user_state.get_glob_storage_name(cluster_name='*')
```
nit: use kwargs for literal.